src/server/ApiRoutes/Hold.js

```diff
@@ -85,6 +85,8 @@ function mapLocationDetails(locations) {
       if (loc['@id'].replace('loc:', '') === c.delivery_location) {
         loc.address = (locationDetails[c.location]) ?
           locationDetails[c.location].address.address1 : null;
+        loc.shortName = (locationDetails[c.location]) ?
```

Comment:
Make `locationDetails[c.location]` a variable so you don't have to call it four times.